Multisig/Logic/Models/Chain.swift

```diff
@@ -45,6 +45,14 @@ extension Chain {
         return chain
     }
 
+    static func by(shortName: String) -> Chain? {
+        dispatchPrecondition(condition: .onQueue(.main))
```

Comment:
why is this precondition necessary?

Reply:
Core Data is not thread safe. We're using the context for the main thread, and so every call to core data or working with any core data object from that context must happen on the same thread.